tests/datasets/test_groupassignee.py

```diff
@@ -165,7 +155,9 @@ def test_messages(self):
         assert ret == InsertBatch([self.DELETED, self.PROCESSED_UPDATE])
 
         assert not message_filter.should_drop(
-            self.__make_msg(0, 42, self.DELETE_MSG, [])
+            self.__make_msg(
+                0, 42, self.DELETE_MSG, [("table", POSTGRES_TABLE.encode())],
```

Comment:
See comment below: https://github.com/getsentry/snuba/pull/1342/files#r490703126

Going to make sure that comment is actually accurate because changing this twice seems less coincidental.

Reply:
Looks like this was just an oversight in the tests and copy/paste error based on the implementation in https://github.com/getsentry/wal2json/pull/7.